src/main/java/com/iota/iri/service/tipselection/impl/WalkerAlpha.java

```diff
@@ -40,10 +49,16 @@ public WalkerAlpha(TailFinder tailFinder, Tangle tangle, MessageQ messageQ, Rand
         this.alpha = config.getAlpha();
     }
 
+    /**
+     * @return {@link WalkerAlpha#alpha}
+     */
     public double getAlpha() {
         return alpha;
     }
 
+    /**
+     * @param alpha {@link WalkerAlpha#alpha}
```

Comment:
I would add a javadoc to `WalkerAlpha#alpha`
Doesn't even need to be mathematical. Can be something along the lines as:
a positive number that controls the randomness of the walk. The closer it is to 0, the rating of the transaction matters less.